lua/orgmode/capture/init.lua

```diff
@@ -419,7 +419,7 @@ function Capture:autocomplete_refile(arg_lead)
 
   if not selected_file then
     return vim.tbl_filter(function(file)
-      return file:match('^' .. vim.pesc(parts[1]))
+      return file:match(vim.pesc(parts[1]))
```

Comment:
```suggestion
      return string.lower(file):match(string.lower(vim.pesc(parts[1])))
```
Fuzzy finding is most convenient, if it's case-insensitive.

Reply:
yes agree, that's even better 👍

Reply:
What about [matchfuzzy](https://neovim.io/doc/user/builtin.html#matchfuzzy()) ?

Reply:
@kristijanhusak Implementing matchfuzzy is different and would take much more of an effort while this simple line of code brings immediately a massive improvement. I would prefer to do experimenting with matchfuzzy in a separate PR. It is also not clear, that it would improve usability, because fuzzy matching can return an unexpectedly large result set and how intuitively it feels depends a lot on the underlying algorithm and implementation.

@kristijanhusak I would prefer to experiment with matchfuzzy in a separate PR. Implementing matchfuzzy is something different and would require a lot more effort, while this simple line of code will immediately bring a massive improvement in usability. I would also like to point out that fuzzy matching does not necessarily improve usability. It can produce an unexpectedly large result set, and how intuitive it feels depends a lot on the underlying algorithm and implementation. Which again raises the effort to implement it correctly, because it needs more experimentation.

Reply:
... I just pushed a version using the `matchfuzzy` 😅 
Doesn't seem to be much effort to implement it. It actually simplified the code imo.
In terms of usability - yes it might yield premature search results. But I don't think that's too bad for usability.

Btw, I found that the first safeguard condition that check the search string is not working.
The `arg_lead` is not nil, it's just an empty string..

Reply:
I take whatever improves the current situation :wink: I was just speaking from my experience with telescope-orgmode, and that it can be hard to fine-tune the result set to something that feels intuitive. And I wasn't sure how motivated you @BreitbandModem would be to change your implementation, because although you modestly state that it wasn't much effort, it's actually a bigger change than just removing a few characters.

Many thanks for the contribution. :+1:

Reply:
Let's go with the fuzzy matching for now, and we can change it later. I'm not using this autocomplete so often,p so if you run into some issues with matching feel free to open up a separate issue.